HHS/Head-Start-TTADP
# TTAHUB-304: Create grantee record page grants widget

## Description of change
Backend changes for Grantee Records page Grants List widget.


## How to test

Browse to the Grantee Records page and verify they grants list reflects the correct list for the Region, Grantee, and Date Range.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-304


## Checklists

### Every PR


~- [ ] Meets accessibility standards (WCAG 2.1 Levels A, AA)~
~- [ ] API Documentation updated~
~- [ ] Boundary diagram updated~
~- [ ] Logical Data Model updated~
~- [ ] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions~

### Production Deploy

~- [ ] Staging smoke test completed~

### After merge/deploy
